### PR TITLE
2680901 Store add link no longer has a destination

### PR DESCRIPTION
**Pull request by haringsrob**

### CODE REVIEW

File: modules/store/commerce_store.module [hunk 1]
```diff
@@ -61,3 +61,32 @@ function template_preprocess_commerce_store(array &$variables) {
...
+/**
+ * Helper function to generate add store links.
+ *
+ * @param array $options
+ *   Array of options to pass to the route generation.
+ *
+ * @return \Drupal\Core\Link
+ *   The link object to be sent to.
+ */
+function _commerce_store_get_add_link($options = []) {
```
Comment (mglaman) [inline modules/store/commerce_store.module]: Is there a way we could always link to .add_page, and have .add_page intelligently redirect to the single form if there is one store?

Comment (haringsrob) [inline modules/store/commerce_store.module]: Looking into this. :)

**Comment by haringsrob** [comment]
Hi,

As discussed with @bojanz , we removed all destination parameters from this. As there is an idea to have a sort of "setup".

**Comment by bojanz** [comment]
Committed in b6e2283ec599ed50da45b5cfbc5e62bdf598e7f7. Thanks!

**Pull request closed without merging**